cr-ideas.md

```diff
@@ -0,0 +1,8 @@
+# Corbin's Random Idea's
+
+- Budget Web App using Plaid's API
```

Comment:
https://youtu.be/ahv9KjJ7t5I

I'm about 20 minutes into this. It looks to be pretty simple to get the integration up and running. I want to recreate and see how easy it would be! Definitely something we could both personally benefit from.